app/action/motivations/post.controller.js

```diff
@@ -0,0 +1,14 @@
+const { body } = require('express-validator')
+
+const validationRules = () => {
+  return [
+    body('motivation')
+      .matches(/^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/, 'i')
```

Comment:
Don't think this is really checking they've selected a motivation - it's checking they've selected one with a UUID we like.